THM-ATLAS/vue-frontend
# Working login page

This adds login api calls and error handling to `/login`.

You can now login with your LDAP credentials and will be redirected to your own user profile. Wrong credentials will be reported as such.

![image](https://user-images.githubusercontent.com/19843204/171721029-7422a41c-83da-48a4-a8c3-3173b416abf7.png)

To test:
* Attempt login with invalid credentials and observe behavior when updating credentials
* Login with valid credentials
* Observe own name on navbar
* Visit `/u/` and observe own profile appearing
* Logout and see that you are indeed logged out

Note: This also disables error popups for internal api errors. Those should now always be handled by the calling component (or in the service if possible)
Note: User data is just stored in localStorage and can be modified by the client.
